### ISSUE TITLE
Chrome doesn't work with Linux subsystem for Windows

### ISSUE THREAD

**Issue by cfjedimaster** [GoogleChrome/lighthouse#3047]
I'm trying to test this under the Windows Subsystem for Linux and get an immediate error:

```
~$ lighthouse https://www.raymondcamden.com --output-path /mnt/c/Users/ray/Desktop/lh.html
Runtime error encountered: Error: The environment variable CHROME_PATH must be set to executable of a build of Chromium version 54.0 or later.
    at Object.linux (/home/ray/.nvm/versions/node/v8.2.0/lib/node_modules/lighthouse/chrome-launcher/chrome-finder.js:107:15)
    at Launcher.<anonymous> (/home/ray/.nvm/versions/node/v8.2.0/lib/node_modules/lighthouse/chrome-launcher/chrome-launcher.js:113:75)
    at Generator.next (<anonymous>)
    at /home/ray/.nvm/versions/node/v8.2.0/lib/node_modules/lighthouse/chrome-launcher/chrome-launcher.js:12:71
    at Promise (<anonymous>)
    at __awaiter (/home/ray/.nvm/versions/node/v8.2.0/lib/node_modules/lighthouse/chrome-launcher/chrome-launcher.js:8:12)
    at Launcher.launch (/home/ray/.nvm/versions/node/v8.2.0/lib/node_modules/lighthouse/chrome-launcher/chrome-launcher.js:98:16)
    at Object.<anonymous> (/home/ray/.nvm/versions/node/v8.2.0/lib/node_modules/lighthouse/chrome-launcher/chrome-launcher.js:42:24)
    at Generator.next (<anonymous>)
    at /home/ray/.nvm/versions/node/v8.2.0/lib/node_modules/lighthouse/chrome-launcher/chrome-launcher.js:12:71
```

Any ideas?

**Comment by patrickhulce** [comment]
I'm not super familiar with how the subsystem for linux works but do you have Chrome installed and accessible from the linux subsystem? What do `which google-chrome` and `which google-chrome-stable` report?

**Comment by cfjedimaster** [comment]
Nothing for both. I do have Chrome installed on Windows of course, but I'd not expect WSL to be able to use it (or... maybe it can?).

**Comment by patrickhulce** [comment]
Yeah I wouldn't either. Looks like there's a thread over in BashOnWindows discussing trying to get Chrome to work over there https://github.com/Microsoft/BashOnWindows/issues/648. Might want to chime in there to see if anyone has current workarounds?

**Comment by cfjedimaster** [comment]
Fair enough - maybe add a note to the readme here just to warn folks? :)

**Comment by paulirish** [comment]
We fixed this a bit ago: https://github.com/GoogleChrome/chrome-launcher/pull/27

now shipped in chrome-launcher 0.8.0 and shipped in lighthouse (2.5.0 definitely, maybe 2.4 too)
